Cloud Application Manager/Deploying Anywhere/using-your-aws-account.md

```diff
@@ -314,6 +317,18 @@ RDS, DynamoDB, and Memcached are CloudFormation boxes. To deploy to an RDS servi
 
 **Note:** If your AWS account has new AMIs, key pairs, security groups, and the like, you must sync with the AWS account in Cloud Application Manager to pick up all the changes.
 
+### Create a Deployment Policy
```

Comment:
This section is already explained in a separate article, please do not duplicate it here.